qemu/tests/cfg/pci_bridge.cfg

```diff
@@ -17,3 +17,42 @@
                     dd_if = ZERO
                 - read:
                     dd_of = NULL
+        - hotplug_unplug:
+            type = pci_bridge_hotplug_unplug
+            disk_driver = "virtio scsi-hd usb2"
+            images += " stg0 stg1 stg2"
+            image_name_stg0 = images/storage0
+            image_name_stg1 = images/storage1
+            image_name_stg2 = images/storage2
+            image_size_stg0 = 1G
+            image_size_stg1 = 1G
+            image_size_stg2 = 1G
+            boot_drive_stg0 = no
+            boot_drive_stg1 = no
+            boot_drive_stg2 = no
+            force_create_image_stg0 = yes
+            force_create_image_stg1 = yes
+            force_create_image_stg2 = yes
+            remove_image_stg0 = yes
+            remove_image_stg1 = yes
+            remove_image_stg2 = yes
+            blk_extra_params_stg0 = "serial=TARGET_DISK0"
+            blk_extra_params_stg1 = "serial=TARGET_DISK1"
+            blk_extra_params_stg2 = "serial=TARGET_DISK2"
+            cmd_timeout = 3600
+            Linux:
+                sub_test_type = dd_test
+                dd_bs = 1K
+                dd_count = 1000
+                chk_usb_cmd = "lsusb -v"
+            Windows:
+                disk_index = "1 2 3"
+                disk_letter = "I J K"
+                chk_usb_cmd = 'wmic path Win32_USBControllerDevice get Dependent | find "USB"'
+                iozone_cmd = "WIN_UTILS:\Iozone\iozone.exe -azR -r 64k -n 125M -g 512M -M -i 0"
+                iozone_cmd += " -i 1 -b %s:\iozone_test -f %s:\testfile"
```

Comment:
why not use sub_test_type "iozone_windows" ?  it include virtio driver installation.  i think the hotpluged disk need driver too.

Reply:
It's not recommended to use "sub_test" and it's ok to ignore driver installation in this case according to @luckyh's opinion. Thanks!

Reply:
ok, got it, it is ok for me.

Reply:
We can remove `-b %s:\iozone_test` or save it in system disk.  As save it in data disk is not very reasonable.

Reply:
Modified, Thanks!